lighthouse-core/audits/user-timings.js

```diff
@@ -78,32 +83,32 @@ class UserTimings extends Audit {
       ut.startTime = (ut.startTime - tabTrace.navigationStartEvt.ts) / 1000;
       if (!ut.isMark) {
         ut.endTime = (ut.endTime - tabTrace.navigationStartEvt.ts) / 1000;
-        ut.duration = ut.endTime - ut.startTime;
+        ut.duration = ut.duration / 1000;
```

Comment:
just style?

Reply:
> just style?

yeah, had to set a `duration` when creating the object, and it seemed silly to set it to 0 only to set it to the actual duration a few lines down..